plugins/openpgp/src/gpgme_helper.vala

```diff
@@ -112,7 +112,13 @@ public static Gee.List<Key> get_keylist(string? pattern = null, bool secret_only
         try {
             while (true) {
                 Key key = context.op_keylist_next();
-                keys.add(key);
+                if (!key.expired && !key.revoked){
```

Comment:
Please fix code style: `if (...) {` and:
``` 
if (...) {
} else {
}
```

Reply:
fixed.